test/contains.js

```diff
@@ -28,6 +28,36 @@ describe('contains', function() {
     assert.strictEqual(R.contains(new Just([42]), [new Just([42])]), true);
   });
 
+  it('dispatches to `indexOf` method', function() {
```

Comment:
Squeamish is a good word for it.  I feel this way too.  It's saying, "Hey, we have a function `foo:: a -> b`, and if you want us to dispatch to your type, then your type needs to implement a method `bar:: obj -> c`."  Something doesn't feel right.